components/CardForm/index.js

```diff
@@ -1,4 +1,4 @@
-import { useRef, useEffect, useState } from "react";
+import React, { useRef, useEffect, useState } from "react";
```

Comment:
We no longer need to import `React`


```suggestion
import { useRef, useEffect, useState } from "react";
```